### DISCUSSION TITLE
Add command doesn't work

### DISCUSSION THREAD

**Discussion by hff399** [shadcn-ui/ui#647] [Q&A]
When I try to add component with `npx shadcn-ui add select`, I get error Invald configuration found in `app/components.json`. There is my components.json - `{
  "$schema": "https://ui.shadcn.com/schema.json",
  "style": "new-york",
  "rsc": true,
  "tailwind": {
    "config": "tailwind.config.js",
    "css": "app/globals.css",
    "baseColor": "gray",
    "cssVariables": false
  },
  "aliases": {
    "components": "@/components",
    "utils": "@/lib/utils"
  }
}`

**Comment by hff399** [comment]
This happened after migration from previous version.

**Reply by zimbavee** [reply]
Try it!
